spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -292,6 +292,19 @@
     and_there_should_be_no_schedule_button
   end
 
+  scenario "Step by step has been updated since it was last checked for broken links" do
```

Comment:
I think this scenario can be greatly reduced. The first 5 steps could be combined into one new one:
`given_there_is_step_by_step_updated_after_links_last_checked`

Reply:
`then_there_should_be_no_inset_prompt` could be used in another test where the step-by-step page is eligible to be published.